docs/reference/esql/esql-across-clusters.asciidoc

```diff
@@ -13,7 +13,7 @@ preview::["{ccs-cap} for {esql} is in technical preview and may be changed or re
 For {ccs-cap} with {esql} on version 8.16 or later, remote clusters must also be on version 8.16 or later.
 ====
 
-With {esql}, you can execute a single query across multiple clusters.
+With <<esql,{esql}>>, you can execute a single query across multiple clusters.
```

Comment:
I think these links to parent pages are less important. would avoid adding them in future